chapter-11/processor/src/main/java/org/acme/Processor.java

```diff
@@ -0,0 +1,20 @@
+package org.acme;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import javax.inject.Singleton;
+import java.net.InetAddress;
+
+@Singleton
+public class Processor {
+  @Incoming("ticks")
+  @Outgoing("processed")
+  Message<String> process(Message<Long> receivedMessage) throws Exception {
```

Comment:
Any reason to use Message instead of payload?

Reply:
Because I was trying to get out metadata at one point but changed tack and forgot to change it to a payload.

Will update